From cefbe17788f847ecbc5e6dce34cf847668b5df58 Mon Sep 17 00:00:00 2001
From: Bartosz Pracz <bartosz.pracz.92@gmail.com>
Date: Tue, 31 Dec 2024 00:29:41 +0100
Subject: [PATCH] all: standardize headers to reflect Git's purpose as a DVCS

Updated headers across multiple files to reflect Git's role as a distributed version control system (DVCS).
This change replaces outdated or informal descriptions, enhancing professionalism and consistency
throughout the codebase.

No functional changes were made, and this commit is purely cosmetic.

Signed-off-by: Bartosz Pracz <bartosz.pracz.92@gmail.com>
---
 Documentation/MyFirstObjectWalk.txt | 2 +-
 builtin/cat-file.c                  | 2 +-
 builtin/check-ref-format.c          | 2 +-
 builtin/commit-tree.c               | 2 +-
 builtin/diff-files.c                | 2 +-
 builtin/hash-object.c               | 2 +-
 builtin/init-db.c                   | 2 +-
 builtin/ls-tree.c                   | 2 +-
 builtin/read-tree.c                 | 2 +-
 builtin/update-index.c              | 2 +-
 builtin/var.c                       | 2 +-
 builtin/write-tree.c                | 2 +-
 config.c                            | 2 +-
 date.c                              | 2 +-
 object-file.c                       | 2 +-
 read-cache.c                        | 2 +-
 t/t4100/t-apply-3.patch             | 4 ++--
 trace.c                             | 2 +-
 usage.c                             | 2 +-
 19 files changed, 20 insertions(+), 20 deletions(-)

diff --git a/Documentation/MyFirstObjectWalk.txt b/Documentation/MyFirstObjectWalk.txt
index dec8afe5b1..ac84a03e17 100644
--- a/Documentation/MyFirstObjectWalk.txt
+++ b/Documentation/MyFirstObjectWalk.txt
@@ -376,7 +376,7 @@ $ ./bin-wrappers/git walken
 
 You should see all of the subject lines of all the commits in
 your tree's history, in order, ending with the initial commit, "Initial revision
-of "git", the information manager from hell". Congratulations! You've written
+of "git", Distributed version control system". Congratulations! You've written
 your first revision walk. You can play with printing some additional fields
 from each commit if you're curious; have a look at the functions available in
 `commit.h`.
diff --git a/builtin/cat-file.c b/builtin/cat-file.c
index b13561cf73..34f4553da4 100644
--- a/builtin/cat-file.c
+++ b/builtin/cat-file.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/check-ref-format.c b/builtin/check-ref-format.c
index cef1ffe3ce..c3e3a6371f 100644
--- a/builtin/check-ref-format.c
+++ b/builtin/check-ref-format.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  */
 #include "builtin.h"
 #include "refs.h"
diff --git a/builtin/commit-tree.c b/builtin/commit-tree.c
index 2ca1a57ebb..7bb4dbb6fa 100644
--- a/builtin/commit-tree.c
+++ b/builtin/commit-tree.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/diff-files.c b/builtin/diff-files.c
index 604b04bb2c..837b9b496a 100644
--- a/builtin/diff-files.c
+++ b/builtin/diff-files.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/hash-object.c b/builtin/hash-object.c
index a25f0403f4..69dcd59333 100644
--- a/builtin/hash-object.c
+++ b/builtin/hash-object.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  * Copyright (C) Junio C Hamano, 2005
diff --git a/builtin/init-db.c b/builtin/init-db.c
index 096f96b9c4..f74b75acb5 100644
--- a/builtin/init-db.c
+++ b/builtin/init-db.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/ls-tree.c b/builtin/ls-tree.c
index 8542b5d53e..3fb4645c73 100644
--- a/builtin/ls-tree.c
+++ b/builtin/ls-tree.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/read-tree.c b/builtin/read-tree.c
index d2a807a828..4571612dd7 100644
--- a/builtin/read-tree.c
+++ b/builtin/read-tree.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/update-index.c b/builtin/update-index.c
index 74bbad9f87..9b57946177 100644
--- a/builtin/update-index.c
+++ b/builtin/update-index.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/builtin/var.c b/builtin/var.c
index 1449656cc9..276aa923c9 100644
--- a/builtin/var.c
+++ b/builtin/var.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Eric Biederman, 2005
  */
diff --git a/builtin/write-tree.c b/builtin/write-tree.c
index 43f233e69b..2d890c55ec 100644
--- a/builtin/write-tree.c
+++ b/builtin/write-tree.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/config.c b/config.c
index 50f2d17b39..e3ff33be04 100644
--- a/config.c
+++ b/config.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  * Copyright (C) Johannes Schindelin, 2005
diff --git a/date.c b/date.c
index a1b26a8dce..a5ad6682f4 100644
--- a/date.c
+++ b/date.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/object-file.c b/object-file.c
index 5b792b3dd4..5e457688f5 100644
--- a/object-file.c
+++ b/object-file.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  *
diff --git a/read-cache.c b/read-cache.c
index 15d79839c2..777a261fa1 100644
--- a/read-cache.c
+++ b/read-cache.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
diff --git a/t/t4100/t-apply-3.patch b/t/t4100/t-apply-3.patch
index cac172e779..08b51e01b3 100644
--- a/t/t4100/t-apply-3.patch
+++ b/t/t4100/t-apply-3.patch
@@ -63,7 +63,7 @@ dissimilarity index 82%
 +++ ls-tree.c
 @@ -1,212 +1,247 @@
 -/*
-- * GIT - The information manager from hell
+- * GIT - Distributed version control system
 - *
 - * Copyright (C) Linus Torvalds, 2005
 - */
@@ -275,7 +275,7 @@ dissimilarity index 82%
 -	return 0;
 -}
 +/*
-+ * GIT - The information manager from hell
++ * GIT - Distributed version control system
 + *
 + * Copyright (C) Linus Torvalds, 2005
 + */
diff --git a/trace.c b/trace.c
index 2cfd25942e..ae43433196 100644
--- a/trace.c
+++ b/trace.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) 2000-2002 Michael R. Elkins <me@mutt.org>
  * Copyright (C) 2002-2004 Oswald Buddenhagen <ossi@users.sf.net>
diff --git a/usage.c b/usage.c
index 47709006c1..af0a32af98 100644
--- a/usage.c
+++ b/usage.c
@@ -1,5 +1,5 @@
 /*
- * GIT - The information manager from hell
+ * GIT - Distributed version control system
  *
  * Copyright (C) Linus Torvalds, 2005
  */
-- 
2.47.0

